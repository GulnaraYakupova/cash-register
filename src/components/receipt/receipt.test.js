import React from 'react';
import Receipt from './receipt';
import Enzyme, { mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

const data = [{
    id: '1',
    name: 'Гречка',
    amount: 2,
    price: 150,
  },
  {
    id: '2',
    name: 'Хлеб',
    amount: 2,
    price: 150,
  },
];

it ('Should render Receipt component', () => {
  const component = mount(
  <Receipt 
    data={data}
    onReceiptItemClick={()=>{}}
    onTotalButtonClick={()=>{}}
   />
  );
  expect(component).toMatchSnapshot();
});

