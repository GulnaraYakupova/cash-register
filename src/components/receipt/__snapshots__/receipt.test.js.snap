// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Receipt component 1`] = `ReactWrapper {}`;
